core/src/mast/node/call_node.rs

```diff
@@ -3,7 +3,8 @@ use core::fmt;
 use miden_crypto::{hash::rpo::RpoDigest, Felt};
 use miden_formatting::prettier::PrettyPrint;
 
-use crate::{chiplets::hasher, Operation};
+use crate::chiplets::hasher;
+use crate::{OPCODE_CALL, OPCODE_SYSCALL};
 
```

Comment:
I noticed that in most of the node files you left this empty line between opcodes import and mast import. Was it intentional?

Reply:
No, actually I was assuming/hoping `rustfmt` was aggressive enough to always give the same kind of formatting.

Fixed in all node files.